.github/workflows/ci.yaml

```diff
@@ -74,15 +74,10 @@ jobs:
         key: ${{ runner.os }}-cargo-build-target-${{ hashFiles('**/Cargo.lock') }}
 
     - name: Compile
-      uses: actions-rs/cargo@v1
```

Comment:
This action creates annotations for errors.

Reply:
yeah, but I personally found that to be distracting rather than valuable